### DISCUSSION TITLE
Sparse Fieldsets support?

### DISCUSSION THREAD

**Discussion by Stiropor** [binaryk/laravel-restify#523] [Q&A]
Is it possible to return only specific fields? Example `https://restify.test/api/restify/articles?fields[articles]=title`?
From what I see json:api specs has this and this would really be useful not to always return all fields.

**Comment by arthurkirkosa** [comment]
Currently this is possible with `related`; will implement this in a future version for main repo as well

**Reply by Stiropor** [reply]
> Currently this is possible with `related`; will implement this in a future version for main repo as well

Yeah, with related it works nicely, only missing this on main query. It's just good not to return all content from backend, smaller load, faster loading. Thank you @arthurkirkosa

**Reply by arthurkirkosa** [reply]
Follow this discussion https://github.com/BinarCode/laravel-restify/discussions/536 ... the point after `policies` is the one that will resolve it.

@binaryk will update each point with a tag number once implemented
